### ISSUE TITLE
Array::Sticky::INC shows wrong distribution

### ISSUE THREAD

**Issue by 2shortplanks** [metacpan/metacpan-web#2113]
Visiting https://metacpan.org/pod/Array::Sticky::INC shows the wrong distribution (Devel-TraceINC rather than Array-Sticky where 02packages places it)

Is this because Devel::TraceINC inlines some of the Array::Sticky::INC and is by the same author?

**Comment by oalders** [comment]
Right, this would be problematic: https://metacpan.org/source/BELDEN/Devel-TraceINC-1.100851/lib/Devel/TraceINC.pm#L100

**Comment by oalders** [comment]
Ping @belden.  :)

**Comment by 2shortplanks** [comment]
Yes, that line should probably be split into two
```
package # hide from the CPAN indexer
  Array::Sticky::INC;
```

**Comment by belden** [comment]
Oh, how annoying for people. [I fixed this 4 years ago](https://github.com/belden/Devel-TraceINC/commit/669edbd497da1f330460f60edb7ea9ed84d34517) but never pushed changes into CPAN.

I've just now pushed a new tarball into PAUSE. `Devel::TraceINC 1.100852` should resolve this.

I'm sorry for my oversight.
